src/lib/firebase.js

```diff
@@ -5,14 +5,16 @@ import 'firebase/firestore';
 
 // Initalize Firebase.
 var firebaseConfig = {
-  apiKey: "AIzaSyAryrdGGzIRpgNIBOYTYm1nbAGP1n60m3E",
-  authDomain: "tcl-16-shopping-list.firebaseapp.com",
-  projectId: "tcl-16-shopping-list",
-  storageBucket: "tcl-16-shopping-list.appspot.com",
-  messagingSenderId: "673795137687",
-  appId: "1:673795137687:web:0cca36b195ad94bd0c89d2"
+  apiKey: 'AIzaSyAryrdGGzIRpgNIBOYTYm1nbAGP1n60m3E',
+  authDomain: 'tcl-16-shopping-list.firebaseapp.com',
+  projectId: 'tcl-16-shopping-list',
+  storageBucket: 'tcl-16-shopping-list.appspot.com',
+  messagingSenderId: '673795137687',
+  appId: '1:673795137687:web:0cca36b195ad94bd0c89d2',
```

Comment:
Similar to previous review comments, I agree that this should be updated down the road to keep secrets out of git. When we take this step we will want to rotate the keys since they've already been committed and will remain in revision history.

Ref: [The 12 Factor App: Store config in the environment](https://12factor.net/config)

Reply:
We typically don't worry too much about these keys because they expire 1 week after the cohort ends. In the meantime, I'll get alerts if our quota usage starts to climb.

Reply:
Sounds good, but I still feel the burden to point out that this is an exception to what we see in production applications! 😄

Reply:
For sure! Just letting everyone know that it's a conscious choice. In most cases, you'd want to keep the keys out of the repo, though because they're client side keys they'd still be discoverable to a determined hacker